script/generate-e2e-tests.js

```diff
@@ -293,6 +293,38 @@ const components = new Map([
           id: 'components-buttongroup-features--icon-buttons',
           name: 'Icon Buttons',
         },
+        {
+          id: 'components-buttongroup-features--icon-buttons-with-tooltip',
```

Comment:
Is this the new way we prefer to create VRTs? I've been manually editing the `.test.ts` files 😅